tictactoe.rb

```diff
@@ -0,0 +1,128 @@
+class Game
+    attr_accessor :board, :board_o, :board_x, :b
+
+    def initialize
+        #to intialize -start method and winning positions
+        start
+       
+        @win_positions =
+      [[0,1,2], [3,4,5], [6,7,8], [0,3,6], [1,4,7], [2,5,8], [0,4,8], [2,4,6]]
+    end
+
+    def start
+        #@turns_played = 0
+         @player = "x"
+        @board = ["-", "-", "-", "-", "-", "-", "-", "-", "-"]
+        @oard_o=0
```

Comment:
There's a typo in "@oard_o", I think you meant "@board_o". This affects your code because "@board_o" instance variable hasn't been defined.